### ISSUE TITLE
Unitless Css Styles should not be converted to string

### ISSUE THREAD

**Issue by Rishabh-Rathod** [postcss/postcss-js#48]
### Version 
"postcss": "^7.0.32",
"postcss-js": "^2.0.3"

#### Expected Behaviour 
```
css -> z-index: 1;
js -> zIndex: 1,
```

#### Actual Behaviour :- 

```
css -> z-index: 1;
js -> zIndex: '1',
```

Css-In-Js libraries like JSS do not support string unitless values.

**Comment by Rishabh-Rathod** [comment]
i have fixed it for my use case let me know if this [_commit_](https://github.com/Rishabh-Rathod/postcss-js/tree/fix-unitless-value) can be merged here too.

**Comment by ai** [comment]
Looks nice. Send PR.

Can you send it today, since we will have a major release today.

**Comment by Rishabh-Rathod** [comment]
> Looks nice. Send PR.
> 
> Can you send it today, since we will have a major release today.

Sure

**Comment by Rishabh-Rathod** [comment]
Issue was resolved in PR #49
